src/server/acl/acl_family.cc

```diff
@@ -308,14 +309,11 @@ void AclFamily::Load(CmdArgList args, ConnectionContext* cntx) {
     return;
   }
 
-  const auto is_successfull = LoadToRegistryFromFile(acl_file, !cntx);
+  const auto has_failed = LoadToRegistryFromFile(acl_file, cntx);
```

Comment:
i think it's less readable. you return the optional result and by reading from the code it's not clear. Lets rename it to `load_error`.

Reply:
btw, I am going to extend ErrorReply to support happy case as well, I do not think `optional` chaining is justified in this case.

Reply:
Please use GenericError then!☝🏻 ErrorReply if for transfering replies, it WAS a capturing reply builder type originally

Reply:
>  I do not think optional chaining is justified in this case.

Given that I can pass it to ReplyBuilder and expect it so send a reply, I think it is. Otherwise, what should it send, OK?

Reply:
i am talking the usage here

Reply:
i renamed it to `load_error` and I changed the return type to `GenericError`